perfect-scrollbar/perfect-scrollbar.d.ts

```diff
@@ -0,0 +1,153 @@
+     * https://github.com/noraesae/perfect-scrollbar#suppressscrollx
+     */
+    suppressScrollX?: boolean;
+
+    /**
+     * When set to true, the scroll bar in Y axis will not be available, regardless of the content height.
+     * https://github.com/noraesae/perfect-scrollbar#suppressscrolly
+     */
+    suppressScrollY?: boolean;
+
+    /**
+     * If this option is true, when the scroll reaches the end of the side, touch scrolling will be propagated to parent element.
+     * https://github.com/noraesae/perfect-scrollbar#swipepropagation
+     */
+    swipePropagation?: boolean;
+
+    /**
+     * When set to true, and only one (vertical or horizontal) scrollbar is visible then both vertical and horizontal scrolling will affect the scrollbar.
+     * https://github.com/noraesae/perfect-scrollbar#usebothwheelaxes
+     */
+    useBothWheelAxes?: boolean;
+
+    /**
+     * If this option is true, when the scroll reaches the end of the side, mousewheel event will be propagated to parent element.
+     * https://github.com/noraesae/perfect-scrollbar#wheelpropagation
+     */
+    wheelPropagation?: boolean;
+
+    /**
+     * The scroll speed applied to mousewheel event.
+     * https://github.com/noraesae/perfect-scrollbar#wheelspeed
+     */
+    wheelSpeed?: number;
+
+    /**
+     * A string.
+     * It's a class name added to the container element.
+     * The class name is prepended with `ps-theme-`.
+     * So default theme class name is `ps-theme-default`.
+     * In order to create custom themes with scss use `ps-container($theme)` mixin, where `$theme` is a scss map.
+     * https://github.com/noraesae/perfect-scrollbar#theme
+     */
+    theme?: string;
+}
+
+interface PerfectScrollbarStatic {
+
+    /**
+     * https://github.com/noraesae/perfect-scrollbar/blob/master/src/js/plugin/initialize.js#L19
+     */
+    initialize(element: HTMLElement, settings?: PerfectScrollbarSettings): void;
+
+    /**
+     * https://github.com/noraesae/perfect-scrollbar/blob/master/src/js/plugin/update.js#L9
+     */
+    update(element: HTMLElement): void;
+
+    /**
+     * https://github.com/noraesae/perfect-scrollbar/blob/master/src/js/plugin/destroy.js#L7
+     */
+    destroy(element: HTMLElement): void;
+}
+
+type JQueryPerfectScrollbarSettingOrCommand = PerfectScrollbarSettings |
+        'update' |
+        'destroy'
+
+interface JQuery {
+
+    /**
+     * https://github.com/noraesae/perfect-scrollbar/blob/master/src/js/adaptor/jquery.js#L6
+     */
+    perfectScrollbar(settingOrCommand?: JQueryPerfectScrollbarSettingOrCommand): JQuery;
+}
+
+interface PerfectScrollbarMountJQuery {
+    (jQuery: JQueryStatic): void;
+}
+
+declare module 'perfect-scrollbar' {
+    const PerfectScrollbar: PerfectScrollbarStatic;
+    export = PerfectScrollbar;
+}
+
+declare module 'perfect-scrollbar/jquery' {
+    const mountJQuery: PerfectScrollbarMountJQuery;
```

Comment:
Since this type is only used once you could simply `declare function mountJquery(jquery: JQueryStatic): void`.